### ISSUE TITLE
Why Tensorflow offcicial implementation doesn't have Validation Set?

### ISSUE THREAD

**Issue by haqkiemdaim** [cr0wley-zz/nmt#3]
Hi there! I just notice that your script was inspired by Tensorflow NMT Implementation.

And if I may ask, why their implementation did not include validation set ya? Like how we suppose to know of our model is overfitting or not?

Like your script got validation set.
